### PR TITLE
Types endpoint

### PR DESCRIPTION
**Pull request by tisto**
Basic implementation of a "@types" endpoint that returns content type schemas as JSON schema.

### CODE REVIEW

**Comment by tisto** [comment]
PR is ready for review. Though, the schema lookup currently fails for Archetype-based content types.

File: docs/source/_json/types_document.json [hunk 1]
```diff
@@ -0,0 +1,1298 @@
...
+      "title": "Exclude from navigation", 
+      "type": "boolean"
+    }, 
+    "expires": {
+      "description": "When this date is reached, the content will nolonger be visible in listings and searches.", 
+      "title": "Expiration Date", 
+      "type": "string"
+    }, 
+    "language": {
+      "choices": [
```
Comment (ebrehault) [inline docs/source/_json/types_document.json]: do we really what that here?

Comment (tisto) [inline docs/source/_json/types_document.json]: Maybe we can reduce the number of available languages in the test fixture to make the auto-generated document more readable.

**Comment by cedricmessiant** [comment]
thanks !

**Pull request merged**